packages/synthetic-chain/src/lib/commonUpgradeHelpers.ts

```diff
@@ -178,7 +197,8 @@ export const voteLatestProposalAndWait = async (
   let { proposals } = (await agd.query('gov', 'proposals')) as {
     proposals: Array<{
       content?: { title: string };
-      messages?: Array<{ '@type': string; content: { title: string } }>;
+      title?: string;
```

Comment:
what added `title` here? please define an interface or union type for what `agd query gov proposals` is returning. would `title` be present when `content` is not?

better yet, use `fetch` to query instead of shelling out to `agd` :https://github.com/Agoric/agoric-sdk/blob/bad9671167b8e063348712cfadcdf581cb06059d/packages/client-utils/src/codegen/cosmos/gov/v1beta1/query.rpc.Query.ts#L68

that returns https://github.com/Agoric/agoric-sdk/blob/master/packages/cosmic-proto/src/codegen/cosmos/gov/v1/gov.ts#L189 but I don't see the two versions defined. is v50 protobuf not yet in master?

Reply:
v50 is not in master yet, and they make a ton of changes to the gov types

Reply:
That said I don't see the relevant changes in https://github.com/Agoric/agoric-sdk/pull/11378/files#diff-48cd0d58e95a9abc7de6265410ba73030c9a5b04bbfdc89a81bbb0c4d21cc4cd either

Reply:
@JeancarloBarrios is doing some tests to confirm, but I believe the issue is a combination of breaking CLI changes, and breaking amino JSON encoding changes that affect RPC responses, combined with possibly a mistake in the initial 0.47 update.

That said, looking at the `.ts` files generated by telescope, I believe it's limited in its handling of "Any" wrapper types, like the one used for the [`messages` field of a proposal](https://github.com/Agoric/agoric-sdk/blob/54fa228dc0a870503752acd18988faf7bea0c38c/packages/cosmic-proto/src/codegen/cosmos/gov/v1/gov.ts#L193C13-L193C16). My understanding is that an RPC response would automatically unwrap the `value` into the corresponding decoded proto message, not leave it as a binary blob. Is that handled at another layer?